src/main/kotlin/blackjack/domain/Hand.kt

```diff
@@ -0,0 +1,24 @@
+package blackjack.domain
+
+class Hand(cardList: List<Card> = emptyList()) {
+    private val cardList = cardList.toMutableList()
```

Comment:
```suggestion
    private val cards = cardList.toMutableList()
```

cardList보다 cards로 이름을 변경하는 건 어떨까요?